sdk/keyvault/Azure.Security.KeyVault.Administration/tests/AdministrationTestBase.cs

```diff
@@ -94,6 +94,13 @@ public sealed override void StartTestRecording()
         {
             base.StartTestRecording();
 
+            // If the AZURE_MANAGEDHSM_URL variable is not defined, we didn't provision one
```

Comment:
Do all administration tests require managed HSM?

Reply:
For the foreseeable future, yes.